free-3.md

```diff
@@ -57,12 +87,19 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I don't think there will be a lot of language decisions -- the input I need only requires 4 basic pieces of information from the user (key signature, clef, tempo in BPM, and time signature). None of the semantics of the language will look very code-y at all! 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I don't think this idea is too ambitious -- it's very easy to determine what keys are pressed and for how long they are pressed, and the time that the key is pressed can be compared to the designated BPM and key signature to determine the duration of the note relative to the BPM (tempo). The tricky part would be to translate that into a visual representation of the notes.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+I think this project really fits the bill of being a domain specific language, and will hopefully provide a helpful abstraction for the process of actually writing out each of the notes and their durations on a musical staff.
+
+This might not be a good idea because it involves writing notes on a staff, which may require my language to communicate with an existing composition software. I can understand how to generate all the notes of a chord (given a root and key signature of a chord, I have a good idea about how to generate the names of the chords because the relationship between the notes in the chord is just dictated by their interval). I think a challenge would be to find a way to visually display the chord on the staff instead of just naming the notes in the chord.
+
```

Comment:
It seems like your DSL is more of a composing assistant, so it doesn't need to generate flawless scores each time.  Therefore, you might not need to deal with chords, but just type the tonic note and have the user come back later and manually fill in the gaps.  On the other hand, I don't see chords as being much more complicated than single notes, at least from the language side, since the user could just hold down multiple keys at once.